src/main/java/io/ebean/enhance/Transformer.java

```diff
@@ -110,42 +133,46 @@ public int getLogLevel() {
 
       boolean isEbeanModel = className.equals(EnhanceConstants.EBEAN_MODEL);
       if (isEbeanModel || enhanceContext.detectEntityTransactionalEnhancement(className)) {
-
-        DetectEnhancement detect = detect(loader, classfileBuffer);
-
-        if (detect.isEntity()) {
-          if (detect.isEnhancedEntity()) {
-            detect.log(3, "already enhanced entity");
-          } else {
-            entityEnhancement(loader, request);
+        try {
```

Comment:
It is not clear to me what this try { ... is here.  What problem is this solving ?

Reply:
When a "NoEnhancementRequiredException" is thrown during "detect", "entiyEnhancement" or "transactionalEnhancement" - the querybean-enhancement was skipped.

Unfortunately, I really don't remember under which constellation this occured here (but we've often the problem, that querybeans were not properly enhanced)